app/controllers/admin/snap_applications_controller.rb

```diff
@@ -17,12 +17,13 @@ class SnapApplicationsController < Admin::ApplicationController
 
     # See https://administrate-prototype.herokuapp.com/customizing_controller_actions
     # for more information
-    #
     def resend_fax
       application = SnapApplication.find(params[:id])
-      Export.create_and_enqueue!(snap_application: application,
-                                 destination: :fax, force: true)
-      flash[:notice] = "Resent fax for #{application.signature}!"
+      Enqueuer.create_and_enqueue_export!(snap_application: application,
+                                          destination: :fax,
+                                          force: true)
+      flash[:notice] = "Resent fax to #{application.office.number} " \
```

Comment:
should we have a test for this? Sometimes I've added flash messages in the wrong place and they don't show up but it's hard to keep track of that without a spec.

Reply:
It depends on how much we want to invest in testing our admin panel; I've been working from the assumption that the admin panel is significantly lower important/priority than the other stuff; so tests covering all the details is not as important; given that it will likely be in significant flux.

That said, I'm always happy to add more tests, especially if someone else is validating their value.